program-overview.md

```diff
@@ -0,0 +1,33 @@
+### OVERVIEW
+Djangonaut Space is a community where open source contributing newcomers, specifically focused on people from historically excluded groups, are welcome and supported to get on their feet to contribute to the Django project.   
+ 
+**Goals**    
+- Increase contribution to and community involvement with Django  
+- Facilitate getting participants started in contributing and guide them to existing resources for continued community and contributions  
+- Provide an actively accessible and inclusive space for mid - higher level Djangonauts.  
```

Comment:
```suggestion
```
I would remove, we have goals in the README which we looked at these for inspiration so I think this is covered